modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx

```diff
@@ -144,21 +148,20 @@ export default function LeftSidebar({
 					type: 'success',
 				});
 			}
-			catch (error) { }
+			catch (error) {}
 		}
 	};
 
 	const onClickGoToFolder = (selectedFolderName: string) => {
-
 		dispatch({
 			payload: {
-				folderName: selectedFolderName.charAt(0).toUpperCase() + selectedFolderName.slice(1),
+				folderName: formatFolderName(selectedFolderName)
```

Comment:
@andremhirai We should probably be using the Folder label here, instead of the name. It would avoid the need of formatting it, also the name is not localizable.

Reply:
@naraandrade when it first loads, it gets the parameter sent by the other application and its passing the name, thats why I left it as is. But I can change it to use the label

Reply:
Yeah, it seems that we're making this mistake in other files as well. It makes sense to pass the folder name or the folder ERC to the Model Builder view, but when we need to display to the user how the folder is called, we should always use the label. Please, talk to Igor about that so we can fix this.